volunteer.html

```diff
@@ -67,6 +67,44 @@ <h3 class="text-center">Development</h3>
     </div>
   </div>
 </div>
+<div class="panel">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center">Testimonials</h2>
+      <div class="row">
+        <div class="column s-4-12">
+          <blockquote class="text-italic">
+            After 1 kickstart event, I was hooked, and haven’t looked back since! After finishing college, I just had to become a blueshirt, to help those like me realise their development potential too.
+          </blockquote>
+          <p class="text-heavy text-center">- Jake Howard</p>
+          <h5 class="text-center">
+            Team CLY SR2014-15<br>
+            Blue Shirt SR2016-21
+          </h5>
+        </div>
+        <div class="column s-4-12">
+          <blockquote class="text-italic">
+            SR was the main reason I signed up to do electronics at university, the experience gained by mentoring and coaching on tech days has been invaluable.
+          </blockquote>
+          <p class="text-heavy text-center">- Tom Wheal</p>
+          <h5 class="text-center">
+            Team PSC SR2014-15<br>
+            Blue Shirt SR2017-21
+          </h5>
+        </div>
+        <div class="column s-4-12">
+          <blockquote class="text-italic">
+            Volunteering with Student Robotics gave me the opportunity to work on projects with real deadlines. When applying for my first professional jobs, that experience let me stand out from other applicants.
```

Comment:
I wrote this? (It's accurate, I just don't remember)
It also feels (to me) a bit dated (the context described is about a decade old). I'm not sure what I'd say instead though. Will have a think and see if there's something I can suggest instead.